pages.ta/linux/aptitude.md

```diff
@@ -0,0 +1,36 @@
+# aptitude
+
+> டெபியன் மற்றும் உபுண்டு தொகுப்பு மேலாண்மை பயன்பாடு
+> மேலும் விவரத்திற்கு: <https://manpages.debian.org/latest/aptitude/aptitude.8.html>.
+
+- கிடைக்கும் தொகுப்புகள் மற்றும் பதிப்புகளின் பட்டியலை ஒத்திசைக்கவும். அடுத்தடுத்த aptitude கட்டளைகளை இயக்கும் முன், இதை முதலில் இயக்க வேண்டும்:
```

Comment:
```suggestion
- கிடைக்கும் தொகுப்புகள் மற்றும் பதிப்புகளின் பட்டியலை ஒத்திசைக்கவும். அடுத்தடுத்த `aptitude` கட்டளைகளை இயக்கும் முன், இதை முதலில் இயக்க வேண்டும்:
```
For commands in Tamil pages I prefer enclosing it in  `` even though the main page doesn't have it.